src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.NetCoreApp.cs

```diff
@@ -113,6 +113,11 @@ public enum SqlConnectionAttestationProtocol
         /// <include file='..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlConnectionAttestationProtocol.xml' path='docs/members[@name="SqlConnectionAttestationProtocol"]/AAS/*' />
         AAS = 1,
 
+#if ENCLAVE_SIMULATOR
+        /// <include file='..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlConnectionAttestationProtocol.xml' path='docs/members[@name="SqlConnectionAttestationProtocol"]/SIM/*' />
+        SIM = 2,
```

Comment:
Is it okay to reserve enum value 2 for SIM?
Does this enum correspond to SQL Server's response as well?

Reply:
This protocol number is defined in SimulatorEnclaveProviders.cs. It is used to create the SqlEnclaveAttestationParameters. I think we need this number to communicate with SQL Server to get a response.

Reply:
My doubt was is it us who own 1 and 3 enum values for supported types (would also answer why 1 and 3 are for AAS/VBS), and if assigning 2 for SIM is also driver specific only?

Does server not care if we assign it 2 or 5 or any other value?

Reply:
I don't think we can randomly pick other numbers as these protocolIDs (AAS / HGS). The protocolID is part of the attestationParameters which the driver needs to be sent to the server through sp_describe_parameter_encryption RPC call. It should be the same numbers across all the drivers targeting the same SQL Server.

Reply:
Resolving based on Offline discussion.